docs/pages/dev/reference/API/org_unit_registry.md

```diff
@@ -0,0 +1,423 @@
+# Prerequisite
+
+## Possible fields
+- `parent_id`
+- `name`
+- `opening_status`
+- `org_unit_type_id`
+- `groups`
+- `geo_point`
+- `shape`
+- `instances`
+
+# POST /api/orgunits/changes/?app_id=..
+
+API used to create or modify an OrgUnit with validation (see feature flags)
+When creating an OrgUnit, the client doesn't know the ID or the OrgUnit was not created yet and we might want to apply 
+some changes still. Therefore, the IDs can be either the DB id or the UUID generated by the client.
+Changes that are not provided are considered not being part of the change request.
+
+## Permissions
+
+User must be authenticated.
+
+## Query parameters
+
+- app_id: String - Must be provided, project for which this is created.
+
+## Body
+
+```json
+{
+  "uuid": "UUID - Client generated UUID",
+  "org_unit_id": "String - id or UUID of the OrgUnit to change, if UUID doesn't exist in DB, it's a new OrgUnit.",
+  "created_at": "Timestamp - format same as /api/instances",
+  "updated_at": "Timestamp - format same as /api/instances",
+  "changes": { "": "Changes is not null and must contain at least one field",
+    "parent_id": "String? - id or UUID of the parent OrgUnit, may be null or omitted.",
+    "name": "String? - Name of the OrgUnit, may be null or omitted.",
+    "opening_status": "Enum<Status>? - One of `open` or `close`",
```

Comment:
There is no `opening_status` field on `OrgUnit.location`. Should we remove this field?

Reply:
Why are you looking for an `opening_status` on the `location` property? I'm confused

Reply:
There is no `opening_status` field on `OrgUnit`. Should we remove this field?

Reply:
I thought it was already there. There is a will to import the status from DHIS2, if I understand correctly, and this would allow us to bring changes to that property. @mestachs might have more information as to where this property currently is.

Reply:
```suggestion
```

We won't implement `opening_status` in V1.